sintr_analysis_server_example/lib/query/completion_metrics.dart

```diff
@@ -5,30 +5,39 @@
 library sintr_worker_lib.completion;
 
 import 'dart:async';
+import 'dart:convert';
+import 'dart:math';
 
 import 'package:sintr_worker_lib/query.dart';
 import 'package:sintr_worker_lib/session_info.dart';
+import 'package:sintr_worker_lib/instrumentation_transformer.dart';
 
 const AVE = 'ave';
 const INCOMPLETE = 'incomplete';
 const MAX = 'max';
 const MIN = 'min';
+const RESPONSE_TIMES = 'responseTimes';
+const RESPONSE_TIME_CLUSTERS = 'responseTimeClusters';
```

Comment:
If I understand correctly these are more buckets than clusters? I.e. they're groups of things of a pre-determined structure that data is added into?